cli/src/main.rs

```diff
@@ -126,6 +129,13 @@ pub fn parse_args<'a>(
         matches.value_of("json_rpc_url").unwrap_or(""),
         &config.json_rpc_url,
     );
+    let rpc_timeout = if let Some(timeout) = matches.value_of("rpc_timeout") {
```

Comment:
You can use clap's `default_value` method to avoid this block: https://docs.rs/clap/2.33.1/clap/struct.Arg.html#method.default_value

Reply:
Using `default_value` now. Much clean!

https://github.com/solana-labs/solana/pull/11181/commits/d3c622b362f85637877dfcb4529c3fbe0c695190#diff-4c438f60247f494e2aee67fcce9e11e0R131-R136